WordPress/gutenberg
# Fix: Align tooltip with others element spacing wise

## What?
This PR is intended to fix: https://github.com/WordPress/gutenberg/issues/64440

## How?
Added specific styles to `Move Up` and `Move Down` Labels

## Testing Instructions
- Create a new post 
- Add a paragraph 
- Notice the direction button label spacing from the Toobar

## Screenshots or screencast 

https://github.com/user-attachments/assets/c3bf3597-5e0e-44d3-a076-5e9ded4a2d28
